pages/deploy/deploy_aws.md

```diff
@@ -518,6 +518,25 @@ diagram.
 With this pipeline we can now deploy the infrastructure to the deployment account(s). Navigate to AWS CodePipeline
 in the tooling account and check the status of your pipeline.
 
+## 12. Setting SES for Email Notifications
+
+Please follow instructions from below if you have enabled email notification by switching the email.active config to true in the `share_notifications` feature under `datasets` module. 
+
+### Moving AWS SES out of Sandbox
+If you have specified `custom_domain`, after the deployment you should see a SES identity which is formed in your AWS Account. 
+You can check it by going to the AWS Console -> AWS SES -> Identities. At this time you have successfully deployed infrastructure to 
+send email notifications via SES, but your AWS account is in the Sandbox mode. When in Sandbox mode, you will have to verify each 
+recipient email id manually. In order to get your SES account out of Sandbox, please follow the instructions in <a href="https://docs.aws.amazon.com/ses/latest/dg/request-production-access.html">Moving out of Sandbox</a> .
+Once your AWS SES account is out of sandbox you can send email to any recipient email id without any prior verification. 
+
+### Monitoring for Email Bounces
+When SES Stack is deployed during the pipeline stage, it will setup a <a href="https://docs.aws.amazon.com/ses/latest/dg/using-configuration-sets.html">configuration set</a>
+which will send any email bounces, delivery failures , rejects & complaints to an SNS topic. In this step, you can add subscriptions to this SNS topic to monitor email delivery problems
+
+In order to do that goto AWS Console -> SNS -> Select the SNS topic which would look like `{resource_prefix}-{envname}-SNS-Email-Bounce-Topic` ( where resource_prefix and envname are specified in the cdk.json ) -> Create Subscription. You can attach multiple subscriptions to 
```

Comment:
typo: "goto" and extra space before comma "failures , rejects"

Reply:
Addressed typos in the next update